inception-kb/src/main/resources/de/tudarmstadt/ukp/inception/kb/model/db-changelog.xml

```diff
@@ -259,5 +259,20 @@
       </column>
     </addColumn>
   </changeSet>
+  
+  <changeSet author="INCEpTION Team" id="20180616-1">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <tableExists tableName="knowledgebase_explicitlyDefinedRootConcepts"/>
+      </not>
+    </preConditions>
+    <createTable tableName="knowledgebase_explicitlyDefinedRootConcepts">
```

Comment:
Please use lowercase table names and separate where necessary using "_".

Reply:
This is the table(name) that hibernate creates when using @**ElementCollection**. 
See comments above.

Reply:
@MdeBoer95 Can you change it, as it will be better if we follow the convention. Or we can discuss it tomorrow if you face any issues with it.